### PR TITLE
update dependencies to solve security issue

### PR DESCRIPTION
**Pull request by mathnogueira**
This PR updates our backend dependencies to patch a security issue with containerd. 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: server/go.mod [hunk 1]
```diff
@@ -29,39 +29,55 @@ require (
...
+	google.golang.org/grpc v1.47.0
 	gopkg.in/yaml.v2 v2.4.0
 )
 
 require (
 	github.com/Microsoft/go-winio v0.5.2 // indirect
-	github.com/containerd/containerd v1.6.2 // indirect
+	github.com/Microsoft/hcsshim v0.9.3 // indirect
+	github.com/containerd/cgroups v1.0.4 // indirect
+	github.com/containerd/containerd v1.6.6 // indirect
```
Comment (mathnogueira) [inline server/go.mod:40]: This is the important update.

File: server/go.sum [hunk 2]
```diff
@@ -2339,6 +2379,8 @@ google.golang.org/grpc v1.42.0/go.mod h1:k+4IHHFw41K8+bbowsex27ge2rCb65oeWqe4jJ5
 google.golang.org/grpc v1.43.0/go.mod h1:k+4IHHFw41K8+bbowsex27ge2rCb65oeWqe4jJ590SU=
 google.golang.org/grpc v1.45.0 h1:NEpgUqV3Z+ZjkqMsxMg11IaDrXY4RY6CQukSGK0uI1M=
 google.golang.org/grpc v1.45.0/go.mod h1:lN7owxKUQEqMfSyQikvvk5tf/6zMPsrK+ONuO11+0rQ=
+google.golang.org/grpc v1.47.0 h1:9n77onPX5F3qfFCqjy9dhn8PbNQsIKeVU04J9G7umt8=
+google.golang.org/grpc v1.47.0/go.mod h1:vN9eftEi1UMyUsIF80+uQXhHjbXYbm0uXoFCACuMGWk=
```
Comment (xoscar) [inline server/go.sum:2383]: Is this `sum`  file similar to the `package-lock.json`?

**Review by xoscar — APPROVED** [review]
LGTM

File: server/go.sum (hunk 2 above)
Comment (mathnogueira) [reply server/go.sum:2383]: Yeah

**Pull request merged**